doc/manuals/admin/yum.md

```diff
@@ -0,0 +1,47 @@
+## Using Yum repositories
+
+This document describes the guidelines of using FIWARE Yum repository to install Orion Context Broker. Provided configurations are corresponds to x86_64 architecture ($basearch) and CentOS/RHEL 7 OS ($releasever)
+
+There are two available packages:
+- nightly, for RPMs which are built from master branch each night (at around 2am).
+- release, for "official" release RPMs. There isn't a fixed released period although it used to be around a release every 1-2 months.
+
+Keep in mind, if you use both repositories together on the same server, nightly builds will always be ahead of release.
+  
+You can manually add a config for yum repositories, that provides nightly and release packages, or download it from FIWARE public repository.
```

Comment:
from the FIWARE public repository

Reply:
fixed in c6e45d153045d7d51e08a4dc3c732985c22fe5c9